autoload/lsp/signature.vim

```diff
@@ -41,6 +41,25 @@ export def InitOnce()
   hlset([{name: 'LspSigActiveParameter', default: true, linksto: 'LineNr'}])
 enddef
 
+def LspShowSignatureCb(timer: number)
+  call g:LspShowSignature()
+enddef
+
+# Checks if the cursor moved over a trigger character to
+# not break autopair plugins
+def DetectJumpOverTriggerChar(triggerChars: list<string>)
+  var col = col('.')
+  var lineStr = getline('.')
+
+  for ch in triggerChars
+    if ch == ','
```

Comment:
What is the need for checking the `,` character here?

Reply:
Without it the signature help will close when you move over the `,` character with the arrow keys. 

See this example:

```c
// while the signature help is open and you are inside the parentheses, after moving over the , 
// from left to right, the signature help would close
printf("val = %d\n, myval);
```

Reply:
Removed